.travis.yml

```diff
@@ -2,33 +2,38 @@ language: python
 dist: xenial
 sudo: required
 python:
- - "3.7"
+- '3.7'
```

Comment:
Just out of curiosity, I see here you change `"` per `'`. But some of the new introduced parameters above are using `"`. Is this intended?

Reply:
Good catch.  I will go over and make it overall consistent now.